extensions/emmet/src/test/wrapWithAbbreviation.test.ts

```diff
@@ -167,7 +167,7 @@ suite('Tests for Wrap with Abbreviations', () => {
 		</div>
 	</a>
 	`;
-		return testWrapWithAbbreviation([new Selection(1, 1, 1, 1)], 'a[href="https://example.com"]>div', expectedContents, contents);
+		return testWrapWithAbbreviation([new Selection(1, 2, 1, 2)], 'a[href="https://example.com"]>div', expectedContents, contents);
```

Comment:
Wondering why these numbers get bumped up. Is it something to do with the indent before each line?

Reply:
`Selection(1, 1, 1, 1)` is when the cursor is before the opening tag
```
	|<a href="https://example.com">
```
And now the check is more strict I moved the cursor inside the tag
```
(startNode.open && startOffset > startNode.open.start && startOffset < startNode.open.end) ||
(startNode.close && startOffset > startNode.close.start && startOffset < startNode.close.end)
```
```
	<|a href="https://example.com">
```

Reply:
Wondering if there's a way to preserve the original behaviour here, maybe by using <= instead of < for the offset checks.

Reply:
yeah it's possible but I think the current behavior is the correct one as it shouldn't include the tag when the cursor is on the left of `<` character
If you prefer the old behavior, let me know and I'll change it

current behavior  | using `<=`
------------ | -------------
![wrap_abbr_exclusive](https://user-images.githubusercontent.com/25115070/108096280-ce8c6700-704e-11eb-86ec-7411e8618677.gif) | ![wrap_abbr_inclusive](https://user-images.githubusercontent.com/25115070/108096305-d5b37500-704e-11eb-9e06-278977f04ad5.gif)

Reply:
That's an interesting case. I guess when the start of the selection is `|<` it should include that node, and if the end of selection is `>|`, it should include that node, but if the end of the selection is `|<`, or if the start of the selection is `>|`, it should not include that node. Is this what your PR currently implements?

Reply:
yeah that's the current behavior